targets/ipq40xx-generic

```diff
@@ -73,6 +76,11 @@ device('gl.inet-gl-b1300', 'glinet_gl-b1300', {
 
 device('linksys-ea6350v3', 'linksys_ea6350v3')
 
+device('linksys-mr8300', 'linksys_mr8300', {
```

Comment:
> How do I remove "dallas" from the expected image name?

Removing "Dallas" *would* best be done in OpenWrt, if needed ( It's not in my opinion). It likely refers to the ARM architecture.
The OpenWrt image is not the one expected to contain `-dallas`, but the one gluon produces is; as the autoupdater looks up [OpenWrts model string for this device](https://github.com/openwrt/openwrt/blob/master/target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-mr8300.dts#L6).

You just add dallas to the devicename like this and should be fine:

```suggestion
device('linksys-mr8300-dallas', 'linksys_mr8300', {
```